_posts/2016-08-01-aquifer.md

```diff
@@ -21,73 +21,73 @@ and in this tutorial I will teach you how by covering:
 
 ## Node
 
-If you don't have `nvm` (node version manager) installed, it is highly
+If you don't have `nvm` ([node version manager](https://github.com/creationix/nvm)) installed, it is highly
 recommended.
 
 ## Installing Aquifer
 
 The Aquifer documentation is officially hosted at:
 [http://aquifer.io/](http://aquifer.io/)
 
-If you are looking to just get Aquifer working the Quick Start Guide will run
-you through the steps of getting Aquifer on to your local environment.
+If you are looking to just get Aquifer working the [Quick Start Guide](http://docs.aquifer.io/sections/quickstart-guide.html) will run
+you through the steps of getting Aquifer set up on your local environment.
 
-WARNING: Until Aquifer releases a new version the version installed following
+WARNING: Until Aquifer releases a new version the version installed when following
 the quick start guide will not work with the rest of this tutorial.
 
-So, which version of Aquifer does work with this tutorial?
+So, which version of Aquifer *does* work with this tutorial?
 
-We will be using the Aquifer branch 1.0.0.  This is the latest development
+We will be using the Aquifer branch 1.0.0. This is the latest development
 branch until we get a full 1.0.0 release.
 
 #### Setting up a development version of Aquifer:
 
-This page contains a section called "Installing for development," this will be
+This page (Which page? Can you specify?) contains a section called "Installing for development," this will be
```

Comment:
@ccjjmartin I was unclear where this "Installing for development" section is located.